keyboards/nibiria/stream15/readme.md

```diff
@@ -0,0 +1,29 @@
+# Acheron Aχξρων 1800CP-SM-S-STM32-MX-TH-WI (codename "Austin") QMK firmware
```

Comment:
This entire file needs to be replaced, as this is lifted straight from another keyboard. You can check `/quantum/template/` for the up to date template for the readme.